### PR TITLE
Add docker compose file.

### PR DESCRIPTION
**Pull request by senpaiSubby**
Added a compose file to allow for dick to insert into docker. Please be gentle.

### CODE REVIEW

**Review by Facinorous-420 — CHANGES_REQUESTED** [review]
Please include a Dockerfile

Thanks for the contribution

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+version: "2.1"
+services:
+  dick:
+    container_name: dick
+    build:
+      context: ./
+      dockerfile: Dockerfile
+    command: npm start
+    volumes:
+      - ./constants.ts:/app/dick/src/constants.ts # DICK constants file
```
Comment (Facinorous-420) [inline docker-compose.yml]: Should be `- ./src/constants.ts:/app/dick/src/constants.ts # DICK constants file`

**Comment by senpaiSubby** [comment]
Docker file has been added good sir.

**Review by Facinorous-420 — APPROVED** [review]
docker-compose and Dockerfile looks good!

**Pull request merged**